### PR TITLE
[SEMI-MODULAR] Reopens #6462

### PR DESCRIPTION
**Pull request by NullDagaf**
## About The Pull Request

Reopens #6462, obliges by maintainers' requests and documents the changes properly.
(I'll fix the merge conflict after making the PR)

## What this PR does

- Bananium golem shells are no longer able to be crafted.
- Kink-Mates are now "illegal". As such. Kink-Mates are removed from public places (not from Maintenance, Ghost Cafe nor DS-1)
- Kink-Mate vending resupplies are now maintenance loot, as well as a purchasable contraband from Cargo.
- Removes some of the more stupid emote sounds, and emotes.
- NSS Journey's pool now uses the pool area.
- Removes some meme shuttles.
- Removes Pierrot's Throat.
- Removes the Bananium Shipment station trait
- Changes the love potion crossbreed to not mention ERP.
- You can no longer obtain bananium via Xenobiology.
- Nor mining.
- Removes the Clown tech node, along with it's respective designs.
- Adds an "Emote Blacklist". If an emote key is written in it's respective text-file, you can't use it intentionally.
- Golden Bike Horns don't flip everyone on screen.
- You can't make a H.O.N.K anymore.

## Why It's Good For The Game

- N/A
- Ever wondered why these are so common, despite numerous "No ERP" posters?
- With that said, hard-removing horny isn't something I want. If you're willing to put in the, frankly, miniscule amount of effort, ye shall have your "fun".
- Requested by a maintainer.
- Map improvement.
- N/A
- N/A
- N/A
- Abandoned crates currently see NO use. This may incentivize people actually going after them.
- OOC meme reference in what's supposed to be informative text.
- N/A
- N/A
- N/A
- Because hard-removing an emote is bad for modularity. This solution's fully modular.
- N/A
- N/A
- N/A

## Changelog
:cl:
del: Bananium golem shells are no longer able to be crafted.
spellcheck: Following a controversy regarding Nanotrasen's "No ERP" policy, kink-mates have been illegalized. 
del: removes some of the more stupid emote sounds, and emotes.
fix: nss journey pool now uses the pool area
del: removes several meme shuttles
del: removes pierrot's throat.
del: removes the bananium shipment station trait
spellcheck: changes the love potion crossbreed to not mention erp
del: you can no longer obtain bananium via xenobiology, nor mining
del: removes the clown tech node
config: adds an "emote blacklist". If an emote key is written in it's respective text-file, you can't use it intentionally.
del: golden bike horns don't flip everyone on screen
/:cl:

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
I actually see a few benefits to most of this. We'll discuss it internally shortly. Could we get bindable emotes perhaps?

**Comment by Gandalf2k15** [comment]
I miss the days of MTA RP where you'd bind an emote and go "Jim Bob flicks his quickdraw holster open, drawing out the X-23 taser gun, aiming it."

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -216,6 +215,8 @@
 	admin_notes = "For player punishment."
 	who_can_purchase = null
 
+//SKYRAT EDIT REMOVAL BEGIN
+/*
```
Comment (Gandalf2k15) [inline code/datums/shuttles.dm]:
```suggestion
```

File: code/datums/shuttles.dm [hunk 2]
```diff
@@ -288,6 +289,8 @@
 /datum/map_template/arena
 	name = "The Arena"
 	mappath = "_maps/templates/the_arena.dmm"
+*/
+//SKYRAT EDIT REMOVAL END
```
Comment (Gandalf2k15) [inline code/datums/shuttles.dm]:
```suggestion
```

File: code/datums/shuttles.dm [hunk 3]
```diff
@@ -194,7 +194,6 @@
 	var/datum/supply_pack/P = SSshuttle.supply_packs[/datum/supply_pack/engineering/shuttle_engine]
 	P.special_enabled = TRUE
 
-
 /datum/map_template/shuttle/emergency/asteroid
```
Comment (Gandalf2k15) [inline code/datums/shuttles.dm:197]: Remove this one though.

File: code/datums/shuttles.dm [hunk 4]
```diff
@@ -360,6 +367,8 @@
 	admin_notes = "An abomination with no functional medbay, sections missing, and some very fragile windows. Surprisingly airtight."
 	movement_force = list("KNOCKDOWN" = 3, "THROW" = 2)
 
+//SKYRAT EDIT REMOVAL BEGIN
+/*
```
Comment (Gandalf2k15) [inline code/datums/shuttles.dm]:
```suggestion
```

File: code/datums/shuttles.dm [hunk 5]
```diff
@@ -370,6 +379,8 @@
 
 /datum/map_template/shuttle/emergency/narnar/prerequisites_met()
 	return SSshuttle.shuttle_purchase_requirements_met[SHUTTLE_UNLOCK_NARNAR]
+*/
+//SKYRAT EDIT REMOVAL END
```
Comment (Gandalf2k15) [inline code/datums/shuttles.dm]:
```suggestion
```

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]

**Comment by ZephyrTFA** [comment]
Atomise literally all of this. This PR is far too spread out

**Comment by Gandalf2k15** [comment]
Modularisation compliance 100%, atomisation 50%

**Comment by Gandalf2k15** [comment]
We'll cherrypick from this.

**Pull request closed without merging**